rig-postgres/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "rig-postgres"
+version = "0.1.0"
+edition = "2021"
+description = "PostgreSQL-based vector store implementation for the rig framework"
+license = "MIT"
+
+[dependencies]
+futures = "0.3.31"
+tokio = { version = "1.43.0", features = ["macros", "rt-multi-thread"] }
+tokio-stream = "0.1.6"
```

Comment:
Are those deps needed outside of tests/examples? I don't see them being used in `lib.rs` so unless I'm missing something, they should be moved to dev dependencies

Reply:
updated dependencies